azure-pipelines.yml

```diff
@@ -264,7 +264,7 @@ stages:
           -w /root/nmodl \
           -v $PWD:/root/nmodl \
           -e NMODL_NIGHTLY_TAG=$TAG \
-          'bluebrain/nmodl:wheel' \
+          'bluebrain/nmodl:wheel-test' \
```

Comment:
TODO: revert this and upload docker image before merge

Reply:
In the end, what changes are needed in the image?